webpack/components/NotificationMenu.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component, PropTypes } from 'react';
+import Notification from './Notification';
+import { readNotification } from '../actions/notification';
+
+export default class NotificationMenu extends Component {
+  constructor(props){
+    super(props);
+
+    this.state = {
+      notifications: props.notifications
+    };
+  }
+
+  notificationClick(id, url) {
```

Comment:
This isn't quite right.  The idea is that as soon as they click on the "Alerts" menu that would set all of the alerts to read.  Basically , the same way facebook does it.